src/main/interface/cli.c

```diff
@@ -3510,6 +3538,84 @@ STATIC_UNIT_TESTED void cliSet(char *cmdline)
     }
 }
 
+#ifdef USE_OSD_PROFILES
+STATIC_UNIT_TESTED void cliOsdProfile(char *cmdline)
```

Comment:
Only use of this function is to unpack simple bitfield ... all other code assumes that user can convert bits to decimal

Reply:
Updated, now works similar to rateprofile, i.e. to check what current osdprofile is or to set it to 0,1,2, or 3.